### PR TITLE
16.1 update

### PR DESCRIPTION
**Pull request by JedMeister**
@OnGle

### CODE REVIEW

File: conf.d/main [hunk 1]
```diff
@@ -28,18 +35,15 @@ service mysql start
...
 
-/var/www/concrete5/concrete/bin/concrete5 c5:install \
+turnkey-concrete5 c5:install \
     --db-server=localhost --db-username=$DB_USER --db-password=$DB_PASS \
-    --db-database=$DB_NAME --site="Turnkey Concrete5" --allow-as-root \
+    --db-database=$DB_NAME --site=TurnKey-Concrete5 --allow-as-root \
     --starting-point=elemental_full --admin-email=$ADMIN_MAIL --admin-password=$ADMIN_PASS
 
-cat >$WEBROOT/.htaccess<<EOF
+cat >> $WEBROOT/.htaccess <<EOF
```
Comment (OnGle) [inline conf.d/main:46]: Is this change from overwrite to appending to the .htaccess file intentional?

**Review by OnGle — COMMENTED** [review]
Just want some clarification.

File: conf.d/main (hunk 1 above)
Comment (JedMeister) [reply conf.d/main:46]: Good catch...

FWIW I actually put that in place to check whether the file was being created or overwritten. Turns out it's being created, so it currently makes no difference. But having said that, I'm not sure whether we should leave it, or change it back to what it was? Thoughts?

Comment (OnGle) [reply conf.d/main:46]: I would've said fix it 6 days ago. But I see no reason to wait more than 6 days, so let's just merge it.

**Pull request merged**